gameci-1_intro.md

```diff
@@ -0,0 +1,146 @@
+When I want to actually deploy to production, I [create a release in GitHub](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release), which will trigger the workflow to run with all the production deployment jobs.
+I can also [manually run the workflow](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow), with the option to input which jobs should run.
+
+The modularity of my workflow is enabled by setting up different triggers and blocking certain jobs with `if`s.
+You may refer to the [complete workflow on GitHub](https://github.com/finol-digital/Card-Game-Simulator/blob/develop/.github/workflows/main.yml), but going forward, I will excerpt the relevant code here.
+
+## The Code
+
+Let's get started with the start of my workflow:
+
+{% highlight yml %}
+{% raw %}
+# .github/workflows/main.yml
+name: Test, Build, and Deploy with GameCI
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - 'Assets/**'
+      - 'Packages/**'
+      - 'ProjectSettings/**'
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
+    paths:
+      - 'Assets/**'
+      - 'Packages/**'
+      - 'ProjectSettings/**'
+  release:
+    types:
+      - published
+  workflow_dispatch:
+    inputs:
+      workflow_mode:
+        description: '[release] [Android, iOS, StandaloneLinux64, WebGL, StandaloneWindows, StandaloneWindows64, WSAPlayer, StandaloneOSX, Steam]'
+        required: false
+        default: ''
+jobs:
+{% endraw %}
+{% endhighlight %}
+
+## Trigger On
+
+As you can see, there are 4 triggers on: `push` to develop, `pull_request` to main, `release` published (with GitHub Release), and `workflow_dispatch`.
+
+The `push` and `pull_request` triggers should run checks to validate each commit, along with a final validation before merging changes to `main`.
+Runs on `push` and `pull_request` should only trigger if a file has changed in the actual Unity project, ie in `Assets/`, `Packages/`, or `ProjectSettings/`.
+Any changes outside of these folders would not cause our Unity project to change, so they can be ignored.
+For example, if we modify the README.md for our project, we wouldn't want the workflow to run, since it wouldn't actually be testing any relevant change to the Unity project.
+
+Runs on `release` will trigger whenever a release is created in the GitHub UI, and we can use `if: github.event.action == 'published'` to detect this scenario.
+
+The triggers for `workflow_dispatch` are the most interesting, since that's where we have setup the most control over the workflow.
+For example, if we wanted to create 2 Windows .exe's to download and run, we can simply pass `StandaloneWindows StandaloneWindows64` as the input.
+This would create both 32-bit and 64-bit Windows executables and upload them to GitHub, where they could then be downloaded for testing.
```

Comment:
```suggestion
This would create both a 32-bit and a 64-bit Windows executable and upload them to GitHub, where they could be downloaded for testing.
```